ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -99,6 +99,15 @@ type opennebula_tm_mad_conf = {
 } = dict();
 
 @documentation{ 
+The following attributes define the default cost for Virtual Machines that don’t have a CPU or MEMORY cost.
```

Comment:
the `'` in `don't` is not a `'` but some othe rfunky character (reported by @StephaneGerardVUB)